qbft/spectest/generate/main.go

```diff
@@ -16,9 +16,13 @@ import (
 //go:generate go run main.go
 
 func main() {
+	clearStateComparisonFolder()
```

Comment:
do we want to make clear the folder in case of panic in the middle? for example if we successfully wrote tests, but could not write state comparison, should we erase all in this case?
this will require `defer recover` or replacing the panics and tracing the error.

Reply:
I personally don't see it as something serious.
The jsons will be in git and we will never merge in something that panics